service/systemd.go

```
@@ -219,14 +219,69 @@ func (j *Systemd) PruneLogs() error {
+		for i, line := range lines {
+			if strings.HasPrefix(strings.TrimSpace(line), "ExecStart=") {
+				// Extract existing command and arguments
+				parts := strings.Fields(strings.TrimPrefix(strings.TrimSpace(line), "ExecStart="))
+
+				// Keep binary path and arguments up to --home
+				newArgs := make([]string, 0)
+				j := 0
+				for ; j < len(parts); j++ {
+					if strings.HasPrefix(parts[j], "--home") {
+						newArgs = append(newArgs, parts[j], parts[j+1])
+						j += 2
```

golangci-lint (1.62.2)

ineffectual assignment to j

(ineffassign)